front-end/src/main/resources/static/index.html

```diff
@@ -26,8 +26,8 @@
 
     <base href="/">
 
-    <script src="https://global.oktacdn.com/okta-signin-widget/4.3.2/js/okta-sign-in.min.js" type="text/javascript"></script>
-    <link href="https://global.oktacdn.com/okta-signin-widget/4.3.2/css/okta-sign-in.min.css" type="text/css" rel="stylesheet"/>
+    <script src="https://global.oktacdn.com/okta-signin-widget/4.4.4/js/okta-sign-in.min.js" type="text/javascript"></script>
```

Comment:
Latest is `v5.2.0` (https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.2.0). Lets upgrade to the latest and the greatest.

Reply:
Updated

Reply:
Updated